pkg/externalresource/serializer.go

```diff
@@ -75,3 +75,7 @@ func (er *ExternalResource) UnmarshalYAML(unmarshal func(interface{}) error) err
 
 	return nil
 }
+
+func (notes *Notes) MarshalYAML() (interface{}, error) {
```

Comment:
This is needed to convert back the external resources Notes from `model.Manifest` to its representation in yaml (a single string)